servo/servo.py

```diff
@@ -219,12 +219,17 @@ class Servo(servo.connector.BaseConnector):
     """The active connectors in the Servo.
     """
 
+    routes: dict[str, type[servo.connector.BaseConnector]] = {}
+    """The dict of connectors this servo was initially configured with. Used for adding connectors that have been removed
+    """
```

Comment:
If CO requests a command cancellation, the servo is restarted in order to kill the currently executing command. This causes the connectors to be removed as they are shutdown and detached to fulfill the event lifecycle (note this may be refactored to only kill tasks rather than shutdown servo altogether but the current form maintains alignment with the previous implementation)

https://github.com/opsani/servox/pull/571/files#diff-7ed9452d2ccb04684818b97a667e535afc48f1ccd8b6f8c903c130d00d94e79bR772-R779

Reply:
Maybe I misunderstood the code, but it seems that 'routes' is used for the initial setup, as well (not only when re-connecting it after it is restarted). That's why I found the comment confusing.

If I got it wrong and this is for re-start only, then maybe: "used for restoring this servo to its initial configuration after it is re-started"

If it is actually used both for start and re-start: "used for adding the connectors to this servo on initialization and on re-start"